openprescribing/api/views_org_details.py

```diff
@@ -31,7 +39,19 @@ def org_details(request, format=None):
         else:
             query += "ORDER BY date, row_id"
     elif org_type == 'ccg':
-        query = 'SELECT pct_id AS row_id, name as row_name, *'
+        if keys:
+            keys = keys.split(",")
+            cols = []
+            aliases = {'row_id': 'pct_id',
+                       'row_name': 'name'}
+            for k in keys:
+                if k in aliases:
+                    cols.append("%s AS %s" % (aliases[k], k))
```

Comment:
We need to pass `keys` as a parameter to `execute_query` rather than inserting directly, to prevent SQL injection attacks.